include/firrtl.xml

```diff
@@ -183,6 +187,15 @@
         <AnyChar String=":," attribute="Separator" context="#stay"/>
         <DetectChar char= ";" context="comment" attribute="Comment"/>
       </context>
+      <context name="enumField" attribute="ID" lineEndContext="#stay">
+        <keyword String="types" attribute="Keyword" context="type"/>
+        <keyword String="outertype" attribute="Keyword" context="outertype"/>
+        <keyword String="typemodifiers" attribute="Keyword" context="#stay"/>
+        <Detect2Char char="{" char1="|" attribute="Separator" context="field"/>
+        <Detect2Char char="}" char1="|" attribute="Separator" context="#pop"/>
+        <AnyChar String=":," attribute="Separator" context="#stay"/>
+        <DetectChar char= ";" context="comment" attribute="Comment"/>
+      </context>
```

Comment:
This looks good to me! Thanks for keeping this esoteric highlighter format up-to-date.